### DISCUSSION TITLE
Nuclei FUZZing

### DISCUSSION THREAD

**Discussion by Sh4d0wHunt3rX** [blacklanternsecurity/bbot#1698] [General]
I was looking a bit on nuclei and thought maybe these are helpful to know for lightfuzz branch.

https://blog.projectdiscovery.io/nuclei-fuzzing-for-unknown-vulnerabilities/
https://docs.projectdiscovery.io/templates/protocols/http/fuzzing-overview

@liquidsec

**Comment by liquidsec** [comment]
Definitely some overlap between what they are doing there and what I am aiming for with lightfuzz.

I guess the biggest difference is they are boiling it down to a template, whereas i'm using python based sub-modules.  

I prefer my approach because there's a finite number of potential things to check for, and they are often extremely complicated (so full-blown python code makes more sense)

Definitely worth looking at to see if I have any gaps what i'm covering
